new-backoffice/extending/modals/README.md

```diff
@@ -78,7 +77,7 @@ A modal can be opened in two ways. Either you register the modal with a route or
 In this case, we use the Modal Token from above, this takes a key as its data. And if submitted then it returns the new key.
 
 ```typescript
-const modalContext = this._modalContext?.open(MY_SOMETHING_PICKER_MODAL, {
+const modalContext = this.#modalManagerContext?.open(MY_SOMETHING_PICKER_MODAL, {
     value: {
         key: this.selectedKey,
```

Comment:
It complains that the Property `'selectedKey' does not exist on type 'MyElement'`

Not sure what is the correct property here 🤔

Reply:
That depends entirely on what the modal accepts - this is just example code so there is no real modal set up to accept anything really. The code is made to illustrate you can set up a value property like this.